Lib/distutils/ccompiler.py

```diff
@@ -148,7 +148,11 @@ class (via the 'executables' class attribute), but most will have:
             if key not in self.executables:
                 raise ValueError("unknown executable '%s' for class %s" %
                       (key, self.__class__.__name__))
-            self.set_executable(key, kwargs[key])
+            if kwargs[key]:
```

Comment:
This could be simplified to `self.set_executable(key, kwargs[key] or None)`.

Reply:
thx. done.